src/java.base/share/classes/jdk/internal/util/Preconditions.java

```diff
@@ -203,13 +203,13 @@ private static String outOfBoundsMessage(String checkKind, List<? extends Number
         // Switch to default if fewer or more arguments than required are supplied
         switch ((args.size() != argSize) ? "" : checkKind) {
             case "checkIndex":
-                return String.format("Index %d out of bounds for length %d",
+                return String.format("Index %s out of bounds for length %s",
                                      args.get(0), args.get(1));
             case "checkFromToIndex":
-                return String.format("Range [%d, %d) out of bounds for length %d",
+                return String.format("Range [%s, %s) out of bounds for length %s",
                                      args.get(0), args.get(1), args.get(2));
             case "checkFromIndexSize":
-                return String.format("Range [%d, %<d + %d) out of bounds for length %d",
+                return String.format("Range [%s, %<s + %s) out of bounds for length %s",
```

Comment:
I assume we have a code coverage here and that we need tests to exercise these code paths.

Reply:
AFAIK, Preconditions was introduced in https://bugs.openjdk.java.net/browse/JDK-8155794

Reply:
It looks like java/util/Objects/CheckIndex should adequately cover this change.